docs/src/index.md

```diff
@@ -0,0 +1,75 @@
+# ManifoldDiffEq
+
+The package __ManifoldDiffEq__ aims to provide a library of differential equation solvers
+on manifolds. The library is built on top of [`Manifolds.jl`](https://github.com/JuliaManifolds/Manifolds.jl) and follows the interface of [`OrdinaryDiffEq.jl`](https://github.com/SciML/OrdinaryDiffEq.jl/).
+
+The code below demonstrates usage of __ManifoldDiffEq__ to solve a simple equation and visualize the results.
+
+Methods implemented in this library are described in, for example:
+
+    > E. Hairer, C. Lubich, and G. Wanner, Geometric Numerical Integration: Structure-Preserving
+    > Algorithms for Ordinary Differential Equations, 2nd ed. 2006. 2nd printing 2010 edition. Heidelberg ;
+    > New York: Springer, 2010.
+
+```julia
```

Comment:
The switch from the reference (that could also be in a footnote?) to the example is a little fast. Could we have an interims section and a short description of the Ode we solve here? Also, since it is a Make example, can we add the result was well?

In the long run either Literate or (my preference) Pluto would be perfect for such examples.

Reply:
I'll make a better introduction, it's just a sketch at the moment. Pluto would be great but I didn't try deploying it yet so I don't know how hard it'd be.